<launch>
    <node name="job_management" pkg="suitbot_ros" type="job_management" output="screen"/>
    <node name="audio_listener" pkg="suitbot_ros" type="audio_listener.py" output="screen"/>
    <node name="audio_speaker" pkg="suitbot_ros" type="audio_speaker.py" output="screen"/>
</launch>